server/src/lib/task.h

```diff
@@ -66,18 +66,18 @@ class Response : public pack::Node
 
         if (status == Message::Status::Ok) {
             if (out.hasValue()) {
-                msg.userData.setString(*pack::json::serialize(out));
+                msg.userData.append(*pack::json::serialize(out));
```

Comment:
hmmm... to be honest, i don't like append here. What if some content already exists in data? Better to make replace here.